packages/runner-ct/src/app/app.tsx

```diff
@@ -40,13 +45,44 @@ const App: React.FC<AppProps> = observer(
     const [pluginsHeight, setPluginsHeight] = React.useState(500)
     const [isResizing, setIsResizing] = React.useState(false)
     const [isSpecsListOpen, setIsSpecsListOpen] = React.useState(true)
+    const [leftSideOfSplitPaneWidth, setLeftSideOfSplitPaneWidth] = React.useState(DEFAULT_LEFT_SIDE_OF_SPLITPANE_WIDTH)
+    const headerRef = React.useRef()
+
+    function monitorWindowResize () {
+      const $header = $(findDOMNode(headerRef.current))
```

Comment:
I think you can just remove find Dom node

Reply:
agreed.